### PR TITLE
UPC-74 Expose SonarLint flag in plugin MANIFEST

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.sonarsource.parent</groupId>
     <artifactId>parent</artifactId>
-    <version>31</version>
+    <version>35</version>
```
Comment (simonbrandhof) [inline pom.xml]: version 35 is buggy. You should downgrade to 34 or wait for 36.

File: sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/PluginManifest.java [hunk 2]
```diff
@@ -77,6 +76,11 @@
    */
   public static final String DEVELOPERS = "Plugin-Developers";
 
+  /**
+   * @since 1.16
+   */
+  public static final String SONARLINT_PLUGIN = "Plugin-SonarLintPlugin";
```
Comment (simonbrandhof) [inline sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/PluginManifest.java]: I suggest value "SonarLint-Supported" to show that it's a boolean.

File: sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/PluginManifest.java [hunk 3]
```diff
@@ -320,6 +326,21 @@ public PluginManifest setUseChildFirstClassLoader(boolean useChildFirstClassLoad
...
+   * @since 1.16
+   */
+  public boolean isSonarLintPlugin() {
+    return sonarlintPlugin;
+  }
+
+  /**
+   * @since 1.16
+   */
+  public PluginManifest setSonarLintPlugin(boolean sonarlintPlugin) {
```
Comment (simonbrandhof) [inline sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/PluginManifest.java]: Same feedback. Method name is strange as it's a boolean. I suggest supportsSonarLint(boolean) and isSonarLintSupported()

Comment (henryju) [inline sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/PluginManifest.java]: setSonarLintSupported for the setter to follow bean convention?

**Comment by simonbrandhof** [comment]
LGTM

**Pull request merged**